lib/components/Upload.js

```diff
@@ -119,6 +119,45 @@ export default class Upload extends Component {
     }
   }
 
+  getDebugLinks(data) {
+
+    if(Array.isArray(data.post_records)) {
+
+      let filename = 'uploader-processed-records.json';
+      let jsonData = JSON.stringify(data.post_records, undefined, 4);
+      let blob = new Blob([jsonData], {type: 'text/json'});
+      let dataHref = URL.createObjectURL(blob);
+
+      let binary_link = null;
+      if(data.pages) {
```

Comment:
Wondering a bit about the nested `if` here. Do you truly want to *not* provide the binary download dump if `data.post_records` isn't there? (Or perhaps the latter is guaranteed to be there at least as an empty array...but still reads a bit funny if there isn't a logical connection, but perhaps there is and I'm missing it.)